pkg/replica/backup.go

```diff
@@ -154,24 +158,33 @@ type BackupStatus struct {
 	BackupURL     string
 	State         ProgressState
 	IsIncremental bool
+	IsOpened      bool
 }
 
-func NewBackup(backingFile *backingfile.BackingFile) *BackupStatus {
+func NewBackup(name, volumeID, snapID string, backingFile *backingfile.BackingFile) *BackupStatus {
```

Comment:
The returned type seems inconsistent with the function. `NewXX` is more used for initializing an instance with type `XX`. In this case, `NewBackup` is more suitable for creating a backup instance instead of backup status. 

Because this is the current code, let's keep it as is for now.